packages/docs/docs/concepts/progressive-loading/stackProgressive.md

```diff
@@ -0,0 +1,109 @@
+---
+id: stackProgressive
+---
+
+# Stack Progressive Loading
+
+Images for the stack viewport can be loaded with a lower resolution/lossy
+version first, followed by increasingly higher resolutions, and finally
+the final version being a lossless representation.
+
+For HTJ2K, this is done automatically when the image is encoded in progressive
+resolution order by using a streaming reader that returns lower resolution versions
+of the image as they are available.
+
+For other image types, a separate lower resolution/lossy version is required.
+The Static DICOMweb toolkit includes some options to create such images.
+
+# Performance
+
+In general, about 1/16-1/10th of the image is retrieved for the lossy/first
+version of the image. This results in about a 10x speed improvement to first
+images. It is affected fairly strongly by overall image size, network performance
+and compression ratios.
```

Comment:
Couple of points:

1. Why can't we explain exactly how much is retrieved first, and why are we giving a range (1/16-1/10)?
2. It is worth mentioning if they can specify this. It reads like we are automatically doing this, although I think the initialBytes is the tool for customizing this.
3. I don't like that we are claiming 10x speed without providing the size. Can you mention the size as well in that sentence?

Reply:
1. seems like it depends on the image data. 64k
2.